provider/haproxy/config/rsyslogd.conf

```diff
@@ -3,15 +3,12 @@ $UDPServerRun 8514
 $template CustomFormat,"%timegenerated% %HOSTNAME% %syslogtag%%msg%\n"
 $ActionFileDefaultTemplate CustomFormat
 
-/* info */
-if $programname contains 'haproxy' and $syslogseverity == 6 then
-    action(type="omfile" file="/var/log/haproxy/traffic")
-
-if $programname contains 'haproxy' and $syslogseverity-text == 'err' then
-    action(type="omfile" file="/var/log/haproxy/errors")
-
-/* notice */
-if $programname contains 'haproxy' and $syslogseverity <= 5 then
-    action(type="omfile" file="/var/log/haproxy/events")
-
-*.* /var/log/messages
+if $programname contains 'haproxy' then {
+  if $syslogseverity == 6 then
+      action(type="omfile" file="/var/log/haproxy/traffic")
+  if $syslogseverity-text == 'err' then
```

Comment:
why are we mix and matching syslogseverity and syslogseverity-text? could we just use syslogseverity?
Is this piece of documentation accurate on the levels? http://www.kiwisyslog.com/help/syslog/index.html?protocol_levels.htm